2019-03-07 13:41:30,697 INFO: Starting Updater
2019-03-07 13:41:30,790 ERROR: Failed to connect org HTTPSConnectionPool(host='api.github.com', port=443): Max retries exceeded with url: /orgs/moodysanalytics (Caused by SSLError(SSLError("bad handshake: SysCallError(10054, 'WSAECONNRESET')")))
